app/src/main/java/net/bible/service/db/readingplan/ReadingPlanRepository.kt

```diff
@@ -63,7 +63,7 @@ class ReadingPlanRepository @Inject constructor() {
     }
 
     @Synchronized
-    fun startPlan(planCode: String, date: Date = CommonUtils.truncatedDate) = scope.launch {
+    fun startPlan(planCode: String, date: Date = CommonUtils.truncatedDate) = runBlocking {
```

Comment:
What does runBlocking here do differently than if we just have it as a ordinary function like

```suggestion
    fun startPlan(planCode: String, date: Date = CommonUtils.truncatedDate) {
```

?

Reply:
It is because it calls the functions `getPlan` and `updatePlan` which are suspending functions so we need the runBlocking block somewhere.

Reply:
And I just follow the style of the other function in `ReadingPlanRepository`

Reply:
Ah correct. Thanks for explanation.